cli/src/web/requests.rs

```diff
@@ -228,6 +229,52 @@ pub fn build_logs(c: &Config, package: &str, build: &Option<String>) {
     }
 }
 
+fn latest_build_logs_quiet(c: &Config, package: &str) -> Option<String> {
```

Comment:
if you want to, you can remove the "Querying server..." message somewhere above, as reasoned for in your comment (yes, I couldn't create a comment up there for some reason)